spring-boot/proxy/README.md

```diff
@@ -6,3 +6,7 @@ This proxy:
 * needs network access to the service API that is to be made accessible 
 * needs to be accessible via the Internet
 
+The gateway-jar file is to be generated by running "coatrack/mvn package". 
+Attempts to package the gateway locally in coatrack/spring-boot/proxy will
+cause a classDefNotFoundException, since classes from coatrack/api are not 
+considered.
```

Comment:
This is not related to the issue that is fixed here. We should discuss this separately. 

Anyway, just f.y.i.: In general, to build only the gateway, which is dependent on the API, the `api` project needs to be installed in the local mvn repository, i.e. use `mvn install` in the api folder first. 

However, best practice in general would be to first build the whole project using the `build.sh` script. To be discussed if that script should also do a `mvn install`.

Reply:
Done.